components/Share/Share.js

```diff
@@ -79,7 +79,7 @@ const Share = props => {
       <List py="xs" space="0">
         <Box as="li">
           <Menu.Link
-            href="#0"
+            cursor="pointer"
```

Comment:
We should probably put `cursor: pointer` as a CSS style in https://github.com/christfellowshipchurch/web-app-v2/blob/main/ui-kit/Menu/Menu.styles.js#L34

Reply:
👍